src/main/webapp/js/disableCategory.js

```diff
@@ -0,0 +1,20 @@
+function disableNow(id) {
+    let httpRequest = new XMLHttpRequest();
```

Comment:
MINOR: como você não muda o valor da variável, dá pra colocar `const` aqui se você quiser.

```suggestion
    const httpRequest = new XMLHttpRequest();
```